docs/source/notebooks/api_quickstart.ipynb

```diff
@@ -16,12 +16,13 @@
      "name": "stdout",
```

Comment:
<p>Maybe use <code>az.summary</code> instead? More used in practice and contains <code>r_hat</code>  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc3/pull/3911/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='74'/>